src/components/Menu.vue

```diff
@@ -34,9 +34,16 @@ export default defineComponent({
 
     onMounted(async () => {
       try {
-        const response = await api.get(
-          '/menus?populate[0]=children&populate[children][populate][0]=children&populate[children][populate][children][populate][0]=children&populate[children][populate][children][populate][children][populate][0]=children&populate[children][populate][children][populate][children][populate][children][populate][0]=children&populate[children][populate][children][populate][children][populate][children][populate][children][populate][0]=children&populate[children][populate][children][populate][children][populate][children][populate][children][populate][children][populate][0]=children&pagination[pageSize]=100'
-        );
+        const query = new URLSearchParams({
```

Comment:
A suggestion: avoid using very long query strings. An alternative is to use URLSearchParams, which is useful both for making them more readable and for constructing them dynamically.

Reply:
Very cool, yes this is smart and makes it much simpler